lib/agoneum/web/controllers/registration_controller.ex

```diff
@@ -6,14 +6,22 @@ defmodule Agoneum.Web.RegistrationController do
 
   action_fallback Agoneum.Web.FallbackController
```

Comment:
I need to figure out if I can add an `only`, `except`, or something similar here that could potentially pick a Fallback based on the format of the request